api/lambdas/sqs-to-converter/index.js

```diff
@@ -342,7 +342,7 @@ export const handler = Sentry.AWSLambda.wrapHandler(async event => {
           await reEnqueue(message);
         } else {
           console.log(
-            `Error processing message: ${JSON.stringify(message)}; ${JSON.stringify(err)}`
+            `Error processing message: ${JSON.stringify(message)}; \n${err}: ${JSON.stringify(err)}`
```

Comment:
Getting `{}` when trying to stringify errors - [link](https://us-west-1.console.aws.amazon.com/cloudwatch/home?region=us-west-1#logsV2:log-groups/log-group/$252Faws$252Flambda$252FSidechainFHIRConverterLambda/log-events/2023$252F06$252F13$252F$255B$2524LATEST$255Da5ff61b85fd54e35a15369163632bf89$3Fstart$3D1686625580597$26refEventId$3D37613007320447546809268952475325893908256697138538086402)

<img width="941" alt="image" src="https://github.com/metriport/metriport/assets/2132564/e474fa1d-6bf7-41d9-b1e1-4ed4d08060f1">